homework-1/student.py

```diff
@@ -0,0 +1,49 @@
+class Student:
+
+    def __init__(self, name, age, id):
+        self.name = name
+        self.age = age
+        self.id = id
+        self.subjects = {}
+
+
+# class CFGStudent(<should inherit from Student>)
+#     create new methods that manage student's subects (add/remove new subject and its graade to the dict)
+#     create a method to view all subjects taken by a student
+#     create a method  (and a new variable) to get student's overall mark (use average)
+
+class CFGStudent(Student):
+
+    def __init__(self, stream, name, age, id):
+        self.stream = stream
+        super(CFGStudent, self).__init__(name, age, id)
+
+    def add_subject(self, subject_grade: dict):
+        self.subjects.update(subject_grade)
```

Comment:
Similarly to the previous exercise, calling the dictionary `student_and_grade` might have been a little clearer here